src/tester.rs

```diff
@@ -475,7 +482,19 @@ fn test_file(
     let num_ignored = Arc::new(AtomicUsize::new(0));
     let pool = ThreadPool::new(inner.test_threads);
     for p in test_files {
-        let test_fname = p.file_stem().unwrap().to_str().unwrap().to_owned();
+        let test_fname = loop {
```

Comment:
`test_file()` is getting pretty long now. Do you think it might be worth breaking this into a helper function?

Reply:
Fixed in https://github.com/softdevteam/lang_tester/pull/58/commits/a658fd03487f922ff73d14c256148f72ed25e3c4.